src/userbase-server/db.js

```diff
@@ -584,8 +606,12 @@ const putTransaction = async function (transaction, userId, databaseId) {
     else throw new Error(`Failed to put transaction with ${e}.`)
   }
 
+  // username is put on the transaction for transmitting,
+  // but not for storing.
+  transaction['username'] = (await userPromise).username
+
   // notify all websocket connections that there's a database change
-  connections.push(transaction, userId)
+  connections.push(transaction)
```

Comment:
I just noticed that Connections.push seems to only accept one argument. So I just removed the second one.